File: AndroidStudioProjects/Basithesapmakinesi/app/src/main/java/com/example/basithesapmakinesi/MainActivity.java
package com.example.basithesapmakinesi;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity
{
    EditText sayı1;
    EditText sayı2;
    TextView sonuc;
    double ilksayi;
    double ikinciSayi;
    double cevap;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        sayı1 = findViewById(R.id.editText3);
        sayı2 = findViewById(R.id.editText4);
        sonuc = findViewById(R.id.textView);
    }
    public void toplama(View view)
    {
        try {
            String ilkRakam = sayı1.getText().toString();//sayı1 stringe çevrildi
            ilksayi = Double.parseDouble(ilkRakam);//stringi int'e çevrildi
            ikinciSayi = Double.parseDouble(sayı2.getText().toString());
            cevap = ilksayi + ikinciSayi;
            sonuc.setText(String.valueOf(cevap));

        }catch (Exception e){
            System.out.println(e + "");
        }
    }


    public void cikarma(View view)
    {
        try {
            ilksayi = Double.parseDouble(sayı1.getText().toString());//strin int'e çevrildi
            ikinciSayi = Double.parseDouble(sayı2.getText().toString());
            cevap = ilksayi - ikinciSayi;
            sonuc.setText(String.valueOf(cevap));
        }catch (Exception e){
            System.out.println(e + "");
        }



    }

    public void carpma(View view)
    {
        try {
            ilksayi = Double.parseDouble(sayı1.getText().toString());//strin int'e çevrildi
            ikinciSayi = Double.parseDouble(sayı2.getText().toString());
            cevap = ilksayi * ikinciSayi;
            sonuc.setText(String.valueOf(cevap));
        }catch (Exception e){
            System.out.println(e + "");
        }
    }

    public void bölme(View view)
    {
        try {
            ilksayi = Double.parseDouble(sayı1.getText().toString());//strin int'e çevrildi
            ikinciSayi = Double.parseDouble(sayı2.getText().toString());
            cevap = ilksayi / ikinciSayi;
            sonuc.setText(String.valueOf(cevap));
        }catch (Exception e){
            System.out.println(e + "");
        }
    }

}

File: AndroidStudioProjects/BasitHesapMakinesi2/app/src/main/java/com/example/basithesapmakinesi/MainActivity.kt
package com.example.basithesapmakinesi

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity()
{


    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun topla(V : View)
    {
        if (sayi1id.text.toString() != "" && sayi2id.text.toString() != "")
        {
            var a  = sayi1id.text.toString()
            var b  = sayi2id.text.toString()
            var sayi :Int = a.toInt()
            var sayi1 :Int = b.toInt()

            textView.text = "Sonuç: ${sayi + sayi1}"

            println("1")



        }
        else if (sayi1id.text.toString() != "" && sayi2id.text.toString() == "")
        {
            var a  = sayi1id.text.toString()

            var sayi :Int = a.toInt()


            textView.text = "Sonuç: ${sayi}"
            println("2")
        }
        else if (sayi1id.text.toString() == "" && sayi2id.text.toString() != "")
        {

            var b  = sayi2id.text.toString()

            var sayi1 :Int = b.toInt()

            textView.text = "Sonuç: ${ sayi1}"
            println("3")
        }
        else if(sayi1id.text.toString() == "" && sayi2id.text.toString() == "" )
        {
            textView.text = "Sonuç: 0"
            println("4")
        }
    }

    fun cikar(v : View)
    {
        if (sayi1id.text.toString() != "" && sayi2id.text.toString() != "")
        {
            var a  = sayi1id.text.toString()
            var b  = sayi2id.text.toString()
            var sayi :Int = a.toInt()
            var sayi1 :Int = b.toInt()

            textView.text = "Sonuç: ${sayi - sayi1}"

            println("1")



        }
        else if (sayi1id.text.toString() != "" && sayi2id.text.toString() == "")
        {
            var a  = sayi1id.text.toString()

            var sayi :Int = a.toInt()


            textView.text = "Sonuç: ${sayi}"
            println("2")
        }
        else if (sayi1id.text.toString() == "" && sayi2id.text.toString() != "")
        {

            var b  = sayi2id.text.toString()

            var sayi1 :Int = b.toInt()

            textView.text = "Sonuç: ${ -1*sayi1}"
            println("3")
        }
        else if(sayi1id.text.toString() == "" && sayi2id.text.toString() == "" )
        {
            textView.text = "Sonuç: 0"
            println("4")
        }

    }
    fun carp(v : View)
    {
        if (sayi1id.text.toString() != "" && sayi2id.text.toString() != "")
        {
            var a  = sayi1id.text.toString()
            var b  = sayi2id.text.toString()
            var sayi :Int = a.toInt()
            var sayi1 :Int = b.toInt()

            textView.text = "Sonuç: ${sayi * sayi1}"

            println("1")



        }
        else if (sayi1id.text.toString() != "" && sayi2id.text.toString() == "")
        {

            textView.text = "Sonuç: 0"
            println("2")
        }
        else if (sayi1id.text.toString() == "" && sayi2id.text.toString() != "")
        {

            var b  = sayi2id.text.toString()

            var sayi1 :Int = b.toInt()

            textView.text = "Sonuç: 0"
            println("3")
        }
        else if(sayi1id.text.toString() == "" && sayi2id.text.toString() == "" )
        {
            textView.text = "Sonuç: 0"
            println("4")
        }

    }
    fun bol(v : View)
    {
        if (sayi1id.text.toString() != "" && sayi2id.text.toString() != "")
        {
            var a  = sayi1id.text.toString()
            var b  = sayi2id.text.toString()
            var sayi :Float = a.toFloat()
            var sayi1 :Float = b.toFloat()

            textView.text = "Sonuç: ${sayi / sayi1}"

            println("1")



        }
        else if (sayi1id.text.toString() != "" && sayi2id.text.toString() == "")
        {


            textView.text = "Sonuç: tanımsız"
            println("2")
        }
        else if (sayi1id.text.toString() == "" && sayi2id.text.toString() != "")
        {

            var b  = sayi2id.text.toString()

            var sayi1 :Int = b.toInt()

            textView.text = "Sonuç: 0"
            println("3")
        }
        else if(sayi1id.text.toString() == "" && sayi2id.text.toString() == "" )
        {
            textView.text = "Sonuç: 0"
            println("4")
        }

    }


}